ceilometer_zenoss/dispatcher/zenoss.py

```
@@ -16,10 +16,34 @@
 #
 ##############################################################################
 
-from oslo.config import cfg
+try:
+    # should work up to Kilo
+    from ceilometer.openstack.common import log
+except ImportError:
+    try:
+        # should work starting from Liberty
+        from oslo_log import log
+    except ImportError:
+        # should not reach here
+        pass
```

can't just swallow the exception- it needs to fail if it can't import it from either location.